docs/upctl_server_network-interface_create.md

```diff
@@ -0,0 +1,38 @@
+## upctl server network-interface create
+
+Create a network interface
+
+```
+upctl server network-interface create <UUID/Title/Hostname...> [flags]
+```
+
+### Options
+
+```
+      --network string         Virtual network ID or name to join.
+      --type string            Set the type of the network. Available: public, utility, private (default "private")
+      --family string          The address family of new IP address. (default "IPv4")
+      --index int              Interface index.
+      --bootable               Whether to try booting through the interface.
+      --source-ip-filtering    Whether source IP filtering is enabled on the interface. Disabling it is allowed only for SDN private interfaces.
+      --ip-addresses strings   Array of IP addresses, multiple can be declared
```

Comment:
not an array. it's a comma-separated list, at least judging by the example below